cmd/Attribution.txt

```diff
@@ -0,0 +1,28 @@
+The following open source projects are referenced by Modbus Device Service Go:
+
+goburrow/modbus (unspecified) - https://github.com/goburrow/modbus
+https://github.com/goburrow/modbus/blob/master/LICENSE
+
+tbrandon/mbserver (MIT) - https://github.com/tbrandon/mbserver
+https://github.com/tbrandon/mbserver/blob/master/LICENSE
+
+BurntSushi/toml (MIT) - https://github.com/BurntSushi/toml
+https://github.com/BurntSushi/toml/blob/master/COPYING
+
+gorilla/mux 1.6.2 (BSD-3) - https://github.com/gorilla/mux
+https://github.com/gorilla/mux/blob/master/LICENSEE
+
+hashicorp/consul 1.1.0 (Modzilla Public License 2.0) - https://github.com/hashicorp/consul
+https://github.com/hashicorp/consul/blob/master/LICENSE
```

Comment:
There's a ton of libraries that this pulls in, and we need to attribute all of them, unless you or someone else can prove that we don't use the code contained in them. Personally, I think it's easier to just attribute all of the dependencies than try to reason about consul's api library... Here's all the libraries I found using godoc.org's imports feature:

* https://github.com/hashicorp/go-cleanhttp
* https://github.com/hashicorp/go-rootcerts
* https://github.com/mitchellh/go-homedir
* https://github.com/mitchellh/mapstructure
* https://github.com/hashicorp/serf (not that we only end up importing the coordinate package here, so I didn't follow the dep chain for the other folders here, so just include the LICENSE for this specific repo)
* https://github.com/armon/go-metrics
* https://github.com/hashicorp/go-immutable-radix
* https://github.com/hashicorp/golang-lru

So just add a line for each library from above.